documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -20,3 +20,4 @@
 ### Changed features
 
 * The `logging.level.com.synopsys.integration` property that was deprecated in [detect_product_long] 9.x, has been removed. Use `logging.level.detect` instead.
+* The FULL_SNIPPET_MATCHING and FULL_SNIPPET_MATCHING_ONLY options for the `detect.blackduck.signature.scanner.snippet.matching` property that was deprecated in [detect_product_long] 9.x, have been removed.
```

Comment:
One suggestion is to remove "that was" before deprecated.
Also thinking, if we change to use [detect_product_short] in these notes, we avoid the "Synopsys vs. Black Duck" 9.x question. (If you know what I mean.)

Thank you.